src/common/Executor.cpp

```diff
@@ -124,12 +124,13 @@ Executor::Executor(int argc, char** argv)
 {
 #if defined(RAJA_PERFSUITE_USE_CALIPER)
   configuration cc;
-  adiak::init(NULL);
-  adiak::user();
-  adiak::launchdate();
-  adiak::libraries();
-  adiak::cmdline();
-  adiak::clustername();
+  #if defined(RAJA_PERFSUITE_ENABLE_MPI)
```

Comment:
@MrBurmark it seems like it may be better to pass the MPI communicator created in main() (RAJAPerfSuiteDriver.cpp) to the Executor ctor and use that where it is needed (e.g., pass to adiak) to ensure consistency. Thoughts?

Reply:
That makes sense to me, though it seems we are using MPI_COMM_WORLD anyway.

Reply:
I realize that. But, I think it's better for consistency. I'll put up a PR after this gets merged.